doc/py_tutorials/py_setup/py_setup_in_ubuntu/py_setup_in_ubuntu.markdown

```diff
@@ -0,0 +1,144 @@
+Install OpenCV-Python in Ubunu {#tutorial_py_setup_in_ubuntu}
+===============================
+
+Goals
+-----
+
+In this tutorial
+    -   We will learn to setup OpenCV-Python in Ubuntu System. Below steps are tested for
+        Ubuntu 16.04 (64-bit) and Ubuntu 14.04 (32-bit).
+
+Introduction:
```

Comment:
header style?

Reply:
Will correct. I referenced Fedora's doc used the previous style while writing this markdown.

Reply:
I see...
Unfortunately it is quite outdated reference document.